src/vue/logo.js

```diff
@@ -0,0 +1,20 @@
+import Vue from '../../node_modules/vue/dist/vue.esm.browser.js';
+
+Vue.component('logo', {
+    props: ['versionConfigurator', 'firmwareVersion', 'firmwareId', 'hardwareId' ],
+    template: `
+        <div class="logo_text">
+            <span>
+                {{ $t("versionLabelConfigurator.message") }}: {{ versionConfigurator }}
+            </span>
+            <br />
+            <span v-if="firmwareVersion && firmwareId">
+                {{ $t("versionLabelFirmware.message") }}: {{ firmwareVersion }} {{ firmwareId }}
+            </span>
+            <br />
+            <span v-if="hardwareId">
+                {{ $t("versionLabelTarget.message") }}: {{ hardwareId }}
+            </span>
+        </div>
+    `,
+})
```

Comment:
Missing semicolon and missing newline
```suggestion
});

```

Reply:
Thanks! Sorry for premature review - I know its a WIP - draft.

Reply:
Yeah, no worries, I'll squash it and tidy up as well. It was more just to see how people react to the idea 🙃